### PR TITLE
W-030904 - Opportunity: Manage Allocations: If your internet is slow or you click the save button twice fast you can end up with over allocations

### PR DESCRIPTION
**Pull request by jbrandolinisf**
# Critical Changes

# Changes
- Modified Manage Allocation Page Buttons to be disabled on click

# Issues Closed

# Community Ideas Delivered

# New Metadata

# Deleted Metadata

### CODE REVIEW

**Comment by jbrandolinisf** [comment]
**lurch: attach W-030904

**Comment by LurchTheButler** [comment]
Tracking <a href="https://foundation.my.salesforce.com/a2x1E000007fQz0QAE" target="_blank">W-030904</a>

File: src/components/UTIL_PageHeader.component [hunk 1]
```diff
@@ -1,68 +1,73 @@
...
-        description="Label for cancel button. When blank will display stgBtnCancel label."/>
+                    description="Label for cancel button. When blank will display stgBtnCancel label."/>
     <apex:attribute name="cancelImmediate" type="Boolean" default="true"
-        description="Whether the cancel action should be immediate, defaults to true."/>
+                    description="Whether the cancel action should be immediate, defaults to true."/>
     <apex:attribute name="cancelDisabled" type="Boolean" default="false"
-        description="Whether the cancel button should be disabled, defaults to false."/>
+                    description="Whether the cancel button should be disabled, defaults to false."/>
+    <apex:attribute name="cancelDisabledOnClick" type="Boolean" default="false"
+                    description="Whether the cancel button should be disabled when clicked/pressed.  Must be rerendered to enable again."/>
```
Comment (force2b) [inline src/components/UTIL_PageHeader.component]: Can you update the descriptions for these to include "Defaults to false" to match the other attributes?

Comment (jbrandolinisf) [reply src/components/UTIL_PageHeader.component]: Done.

**Comment by markross-sfdo** [comment]
@jbrandolinisf Could you please make a GitHub issue for this? Per NPSP's practices, we need to add an Issue to the NPSP release notes instead of Changes text. Since this is already merged, you'll have to manually close it too. Please let me know when it's created so we can add it to the Release Notes Drafts. Thank you!

**Comment by jbrandolinisf** [comment]
@markross-sfdo - It is created, linked and closed.  See #5335

**Pull request merged**